src/components/Progress/types.ts

```diff
@@ -10,3 +10,9 @@ export interface TProgressPropTypes {
   loop?: boolean
   hasError?: boolean
 }
+
+export interface TProgressBarPropTypes {
+  stepItems?: TProgressPropTypes[]
```

Comment:
lets make `stepItems` mandatory

Reply:
also write here `TProgressPropTypes & { id: string | number }[]` and then in ProgressBar.tsx file you will use id instead of index